### ISSUE TITLE
Remove CGI::Switch from Kernel/cpan-lib

### ISSUE THREAD

**Issue by bschmalhofer** [RotherOSS/otobo#1900]
The bundled version of CGI is 4.36. But there is no CGI::Switch in CGI 4.36.
The file _Kernel/cpan-lib/CGI/Switch.pm_  had been added as part of CGI 4.04 in 2014. So it is safe to remove that file. Let's do that in rel-10_1.

_Originally posted by @bschmalhofer in https://github.com/RotherOSS/otobo/issues/1878#issuecomment-1231761342_

**Comment by bschmalhofer** [comment]
_Kernel/cpan-lib/CGI/Switch.pm_ is removed. Closing this issue.

**Comment by bschmalhofer** [comment]
Fixed the milestone to 10.1.6, because that is where CGI::Switch has been removed.

```
bernhard@bernhard-Aspire-A515-57:~/devel/OTOBO/otobo$ git checkout rel-10_1_5
Vorherige Position von HEAD war 9664befc89 Create ARCHIVE. New OTOBO release: 10.1.6
HEAD ist jetzt bei fe332bc910 Create ARCHIVE. New OTOBO release: 10.1.5
bernhard@bernhard-Aspire-A515-57:~/devel/OTOBO/otobo$ ls Kernel/cpan-lib/CGI/Switch.pm 
Kernel/cpan-lib/CGI/Switch.pm
bernhard@bernhard-Aspire-A515-57:~/devel/OTOBO/otobo$ git checkout rel-10_1_6
Vorherige Position von HEAD war fe332bc910 Create ARCHIVE. New OTOBO release: 10.1.5
HEAD ist jetzt bei 9664befc89 Create ARCHIVE. New OTOBO release: 10.1.6
bernhard@bernhard-Aspire-A515-57:~/devel/OTOBO/otobo$ ls Kernel/cpan-lib/CGI/Switch.pm 
ls: Zugriff auf 'Kernel/cpan-lib/CGI/Switch.pm' nicht möglich: Datei oder Verzeichnis nicht gefunden
```
